Cabal/src/Distribution/Simple/Flag.hs

```diff
@@ -67,6 +67,14 @@ instance Applicative Flag where
   NoFlag   <*> _ = NoFlag
   pure = Flag
 
+instance Foldable Flag where
```

Comment:
These two instances should be derived instead perhaps.

Reply:
👍 Done.